docs/_data/nakadi-event-bus-api.yaml

```
@@ -3110,6 +3122,52 @@ definitions:
         example: 'data.security.team'
 
 
+  Annotations:
+    type: object
+    description: >
+      # [Work in progress] Annotations of the Nakadi resource.
+
+      A Nakadi event-type or a subscription is considered as a Nakadi resource. A Nakadi resource can have a set of
+      annotations associated with it. The syntax of annotations is adopted from Kubernetes.
+
+        From the Kubernetes documentation about annotation keys:
+      > Annotations are key/value pairs. Valid annotation keys have two segments: an optional prefix and name, separated
+      by a slash (`/`). The name segment is required and must be 63 characters or less, beginning and ending with an
+      alphanumeric character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`), dots (`.`), and alphanumerics
+      between. The prefix is optional. If specified, the prefix must be a DNS subdomain: a series of DNS labels
+      separated by dots (`.`), not longer than 253 characters in total, followed by a slash (`/`).
+
+        Values of the annotations are currently limited to 1000 bytes. This can be relaxed in the future.
```

```suggestion
      Values of the annotations are currently limited to 1000 bytes. This can be relaxed in the future.
```
Feels like this should be indented at the same depth as the first paragraph above.  See also "From the K8s docs..."

---

This is an intentional to make it render properly in the API portal.

---

Ah, where can I see it?